shell/TestPackage.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$LastExitCode = 0
+$env:PATH +=";$($env:SystemDrive)\\ProgramData\\chocolatey\\bin"
+
+# https://github.com/chocolatey/choco/issues/512
+$validExitCodes = @(0, 1605, 1614, 1641, 3010)
+
+$package  = ls c:\packages\*.nupkg
```

Comment:
I may have missed that - I saw you split the name of the file, but I may have missed where you chose only the first package in the directory.